ProPublicaCongressAPI/ProPublicaCongressAPI.csproj

```diff
@@ -13,7 +13,6 @@
     <PackageReference Include="Newtonsoft.Json" Version="10.0.2" />
   </ItemGroup>
   <ItemGroup>
-    <Folder Include="InternalModels\" />
```

Comment:
Apologies.  I'm using a flighted version of VS and it has a bug where it keeps removing this line.  Fixed now.